src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -410,8 +409,13 @@ protected virtual void Initialize()
                 }
 
                 // Load builtin extensions
-                var sendGridExtension = new Extensions.SendGrid.SendGridConfiguration();
-                LoadExtension(sendGridExtension);
+                {
+                    var botExtension = new Extensions.BotFramework.Config.BotFrameworkConfiguration();
```

Comment:
Yes. They have unit tests, which I've verified. This happens in the new ApplyConfig()  in their extension.